examples/spring-data-proxy/README_pt.adoc

```diff
@@ -0,0 +1,18 @@
+:index-group: Frameworks
+:jbake-type: page
+:jbake-status: status=published
+
+== Amostra Spring Data
+
+Este exemplo usa ganchos OpenEJB para substituir uma implementação EJB por um 
+proxy usa o Spring Data no seu contêiner preferido.
```

Comment:
in the sentence "implementação EJB por um proxy usa o Spring Data", i think that "implementação EJB por um proxy para usar o Spring Data" make more sense.

Reply:
done thanks marco : )